error/README.md

```diff
@@ -3,10 +3,11 @@ Please don't use it directly but rely on `twurst-client` or `twurst-server`that
 
 ## Cargo features
 - `serde` allows to (de)serialize the error using [Serde](https://serde.rs/) following the official Twirp serialization.
-- `http` allows to convert between [`http::Response`](https://docs.rs/http/latest/http/response/struct.Response.html) objects and Twirp errors,
+- `http` allows to convert between [`http::Response`](https://docs.rs/http/1/http/response/struct.Response.html) objects and Twirp errors,
```

Comment:
Should we add a comment in the Cargo.toml to remember to update this to match the version there ?

Reply:
Done. Great call!